editor/gui/scene_tree_editor.h

```diff
@@ -82,7 +104,9 @@ class SceneTreeEditor : public Control {
 
 	void _compute_hash(Node *p_node, uint64_t &hash);
 
-	void _add_nodes(Node *p_node, TreeItem *p_parent);
+	void _update_nodes(Node *p_node, TreeItem *p_parent, bool force_update = false);
+	void _update_node(Node *p_node, TreeItem *p_item, bool part_of_subscene);
```

Comment:
It would be nice if these methods' names weren't that much similar.

Reply:
_update_node_tree() and _update_node() ?

Reply:
More like `_update_node_sub_tree()` or `_update_node_branch()`.